lib/blemulator/simulation/simulated_characteristic.dart

```diff
@@ -13,7 +13,7 @@ class SimulatedCharacteristic {
   bool isNotifying;
   bool isIndicatable;
 
-  StreamController<Uint8List> _streamController;
+  StreamController<Uint8List> streamController;
```

Comment:
This should stay private and you should instead use write() to add values to stream.